org.srb2.SRB2Kart-galaxy_azerty.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2023 Flathub maintainers -->
+<component type="desktop-application">
+  <id>org.srb2.SRB2Kart-galaxy_azerty</id>
```

Comment:
since it's a mod, is this still "controlled by the upstream developers" of the mod?

Reply:
SRB2Kart is already a mod of a mod. The developpers allow forks.
SRB2Kart Galaxy is a mod of this released within the ecosystem of the SRB2Kart community. On the official motherboard a mod dev specify wether or not someone can re-use their work.
For 'galaxy' mod, the developper maked his mod as "reusable" with the following caption:
> Modify: YES - Maintain: YES - I give permission for my entire submission to be modified by others or used in their own work. I give permission for my entire submission to be maintained by others as well.

([source](https://mb.srb2.org/addons/srb2kart-galaxy.4500/))

So I take this shouldn't be a problem.